documentation/modules/auxiliary/scanner/msmail/user_enum.md

```diff
@@ -0,0 +1,18 @@
+OWA (Outlook Webapp) is vulnerable to time-based user enumeration attacks.
+ This module leverages all known, and even some lesser-known services exposed by default
+ Exchange installations to enumerate users. It also targets Office 365 for error-based user enumeration.
+
+**Userenum (o365) Command**
+- Error-based user enumeration for Office 365 integrated email addresses
+
+**Note:**  Currently uses RHOSTS which resolves to an IP which is NOT desired, this is currently being fixed 
+
+## Verification
```

Comment:
replace these with some redacted test runs, so someone looking at the docs later would see what the output of this module should look like